recipes/mysql-concpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.25)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```
Why not the default of CMake 3.15?